docs/examples_google_research_examples.rst

```diff
@@ -0,0 +1,258 @@
+
+  - `Performer's Fast Attention (FAVOR+) module <https://github.com/google-research/google-research/tree/master/performer/fast_attention>`__
+
+- Research paper:
+
+  - `Rethinking Attention with Performers <https://arxiv.org/abs/2009.14794>`__ (Choromanski et al., 2020)
+
+    - Introduces *"Performers, Transformer architectures which can estimate regular (softmax) full-rank-attention Transformers with provable accuracy, but using only linear (as opposed to quadratic) space and time complexity, without relying on any priors such as sparsity or low-rankness. To approximate softmax attention-kernels, Performers use a novel Fast Attention Via positive Orthogonal Random features approach (FAVOR+), which may be of independent interest for scalable kernel methods. FAVOR+ can be also used to efficiently model kernelizable attention mechanisms beyond softmax."*
+
+Self-attention Does Not Need O(n^2) Memory
+==========================================
+
+- `Code on GitHub <https://github.com/google-research/google-research/tree/master/memory_efficient_attention>`__
+- `Colab notebook <https://github.com/google-research/google-research/blob/master/memory_efficient_attention/memory_efficient_attention.ipynb>`__
+
+- Research paper:
+
+  - `Self-attention Does Not Need O(n^2) Memory <https://arxiv.org/abs/2112.05682>`__ (Rabe and Staats, 2021)
+  
+    - *"We present a very simple algorithm for attention that requires O(1) memory with respect to sequence length and an extension to self-attention that requires O(log n) memory. This is in contrast with the frequently stated belief that self-attention requires O(n^2) memory. While the time complexity is still O(n^2), device memory rather than compute capability is often the limiting factor on modern accelerators. Thus, reducing the memory requirements of attention allows processing of longer sequences than might otherwise be feasible..."*
+
+Computer vision
+***************
+
+Colorization Transformer (ColTran)
+==================================
+
+- `Code on GitHub <https://github.com/google-research/google-research/tree/master/coltran>`__
+
+- Research paper:
+
+  - `Colorization Transformer <https://openreview.net/forum?id=5NA1PinlGFu>`__ (Kumar et al., 2020)
+
+    - *"We presented the Colorization Transformer (ColTran), an architecture that entirely relies on selfattention for image colorization. We introduce conditional transformer layers, a novel building block for conditional, generative models based on self-attention. Our ablations show the superiority of employing this mechanism over a number of different baselines. Finally, we demonstrate that ColTran can generate diverse, high-fidelity colorizations on ImageNet, which are largely indistinguishable from the ground-truth even for human raters."*
+
+Vision Transformer (ViT) and MLP-Mixer Architectures
+====================================================
+
+- `Code on GitHub <https://github.com/google-research/vision_transformer>`_
```

Comment:
You could also add https://github.com/google-research/big_vision as well, since that's the codebase that was used to train these models. The codebase you linked is the canonical codebase for fine-tuning and inference with these models.

Reply:
Thank you @andsteing . Done